x-pack/test/security_solution_api_integration/test_suites/genai/attack_discovery/schedules/utils/helpers.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ToolingLog } from '@kbn/tooling-log';
+import type SuperTest from 'supertest';
+
+import {
+  API_VERSIONS,
+  ATTACK_DISCOVERY_SCHEDULES_ALERT_TYPE_ID,
+} from '@kbn/elastic-assistant-common';
+import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';
+import {
+  countDownTest,
+  routeWithNamespace,
+} from '../../../../../../common/utils/security_solution';
+import { getAttackDiscoverySchedulesApis } from './apis';
+import { getSimpleAttackDiscoverySchedule } from '../mocks';
+
+export const enableAttackDiscoverySchedulesFeature = async (supertest: SuperTest.Agent) => {
+  const response = await supertest
+    .put('/internal/core/_settings')
+    .set('elastic-api-version', '1')
+    .set('kbn-xsrf', 'some-xsrf-token')
+    .set('x-elastic-internal-origin', 'kibana')
+    .send({
+      'feature_flags.overrides': {
+        'securitySolution.assistantAttackDiscoverySchedulingEnabled': true,
```

Comment:
Sweet utility function!
This call will be made redundant when https://github.com/elastic/kibana/pull/228327 merges, but this pattern is useful